packages/panels/src/Pages/Dashboard/Actions/FilterAction.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Filament\Pages\Dashboard\Actions;
+
+use Exception;
+use Filament\Actions\Action;
+use Filament\Pages\Dashboard;
+use Filament\Support\Facades\FilamentIcon;
+use Livewire\Component;
+
+class FilterAction extends Action
+{
+    public static function getDefaultName(): ?string
+    {
+        return 'filter';
+    }
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->label(__('filament-panels::pages/dashboard.actions.filter.label'));
+
+        $this->modalHeading(fn (): string => __('filament-panels::pages/dashboard.actions.filter.modal.heading'));
+
+        $this->modalSubmitActionLabel(__('filament-panels::pages/dashboard.actions.filter.modal.actions.apply.label'));
+
+        $this->icon(FilamentIcon::resolve('panels::pages.dashboard.actions.filter.grouped') ?? 'heroicon-m-funnel');
```

Comment:
Should this be grouped, or should the alias be updated?

Reply:
Updated the alias to remove grouped